### ISSUE TITLE
[Bug] FastForward MV3 on Firefox broken after AMO relisting

### ISSUE THREAD

**Issue by Cynosphere** [FastForwardTeam/FastForward#1130]
### Body

FastForward does not work at all after updating after FastForward being relisted on AMO

### Version

Manifest-v3

### What browser(s) are you seeing the problem on?

Firefox browsers

### What platform are you seeing the problem on?

Computer (Windows, Linux, MacOS)

### Anything else?

I am not joining your Discord. Stop contributing to information voiding.

**Comment by runxel** [comment]
Can confirm. The new version does nothing.

**Comment by MarkJGx** [comment]
What's "information voiding"?

**Comment by driedpampas** [comment]
We are currently investigating this (and another issue with the version not showing up)

**Comment by lostdusty** [comment]
We are aware of the issue already, and trying to get a fix for this.

Could you explain "information voiding", please?

**Comment by Cynosphere** [comment]
http://ascii.textfiles.com/archives/5509

**Comment by lostdusty** [comment]
..what? we have documentation on github wiki, and we are working on the fix, but we didnt comment about anything with the public, until this issue, because we thought this was going to be a simple fix, but the way mozilla wants the consent dialog to be implemented make it a hard thing to fix

**Comment by undeadjess** [comment]
update:
this is happening because Mozilla asked us to add a consent dialogue in the new version because of new rules they have about processing urls. they said we have to have the user press agree BEFORE running any code. unfortunately, the code that manages the consent page isn't working properly, so the extensions main code (the background script) isn't being run, so nothing is working.

#1134 may fix this, but needs testing. once that is done, we will upload the new fixed version to Firefox and hopefully this issue will be sorted

**Comment by driedpampas** [comment]
update: 
We've identified possible problematic commits going back quite a bit

**Comment by driedpampas** [comment]
another update:
we've identified the problematic code and are testing a fix

**Comment by escape0707** [comment]
So, unless a new release is published to AMO, Firefox users still can't use this addon unless they install it manually somehow?

If so, is there any instruction about how to manually install git version of this addon?

**Comment by driedpampas** [comment]
exactly or you can download the latest package zip file in actions (and if using firefox stable or beta, change the contact email in the manifest and upload it to firefox addons as self distributed so they sign it real quick so you can install it)

**Comment by escape0707** [comment]
As of recent update, it's working now.

(Although some Example Links on the homepage can't be bypassed, like the [clictune.com](https://www.clictune.com/ix7b) one. But I think that belongs to another issue
